src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -32,10 +49,71 @@ const ChangeRequestsTable: React.FC = () => {
 
   const theme = useTheme();
 
+  useEffect(() => {
```

Comment:
is this the normal way to track window resizing in react?

Reply:
yeah this is what I found, you need the useEffect/ listener so that it updates in real time